iron-overlay-manager.html

```diff
@@ -43,8 +43,8 @@
     // NOTE: Use useCapture=true to avoid accidentally prevention of the closing
     // of an overlay via event.stopPropagation(). The only way to prevent
     // closing of an overlay should be through its APIs.
-    Polymer.Gestures.add(document, 'tap', null);
-    document.addEventListener('tap', this._onCaptureClick.bind(this), true);
+    Polymer.Gestures.add(document.documentElement, 'tap', null);
```

Comment:
Here an example showing the possible regression we might introduce by changing line 47
http://jsbin.com/qubifad/1/edit?html,console,output